### DISCUSSION TITLE
Unable to Access Web Admin Panel After Version Update

### DISCUSSION THREAD

**Discussion by mesuttavukcu** [rustdesk/rustdesk-server-pro#496] [General]
Hello,

After updating my RustDesk Pro server to the latest version, I am unable to access the Web Admin Panel. When I try to log in, only a blank white screen appears.

The update was completed successfully.
I have cleared my browser cache and tried accessing the panel using different browsers.
The server logs do not show any errors related to this issue.
Could you please help me resolve this issue? Are there any specific troubleshooting steps or configurations I need to check?

Thank you in advance for your assistance.

**Comment by rustdesk** [comment]
@fufesou

**Comment by fufesou** [comment]
@mesuttavukcu Hi, could you please

1. Enter the login page
2. Press "F12"
3. Switch to the "Console" tab
4. Login
5. Check if there're any errors other than messages like `Missing message: "..." for locale: "en-US", using default message as fallback.`

You can clear messages before login.

![a](https://github.com/user-attachments/assets/bdd4322f-d589-4dc2-9782-30682478fea1)

**Reply by fufesou** [reply]
You can also check the "Console" messages if the login page is blank.

**Reply by mesuttavukcu** [reply]
console is empty cannot see login screen 
http://rust.data-net.com.tr:21114/static/index.html 
please see my page

**Reply by fufesou** [reply]
It seems some files are missing.

Could you please run `ls /usr/share/rustdesk-server` and `/usr/share/rustdesk-server/static` on the server machine.

**Reply by fufesou** [reply]
You may need to reinstall hbbs if the fiels are missing.

But did you encounter any errors during the upgrade process?
This error is very strange.

**Reply by mesuttavukcu** [reply]
There is no such folder on the server, it did not give any errors during the upgrade.

**Comment by mesuttavukcu** [comment]
![image](https://github.com/user-attachments/assets/c308f661-219a-4b3c-95c0-28d6d90c6eb6)

**Reply by fufesou** [reply]
Thanks.

Could you please try

1. `mkdir -p /usr/share/rustdesk-server && cp -r /var/lib/rustdesk-server/static /usr/share/rustdesk-server`
2. Check if the login page is ok

You can directly run `rm -rf /usr/share/rustdesk-server` to remove the copied files after the test.

**Reply by fufesou** [reply]
Sorry, the path in the command is wrong. The source dir is `/var/lib`, I've changed it in the comment.

The files are in `/usr/share/rustdesk-server` on my machine.

**Reply by fufesou** [reply]
The files are all `0B`?

You may also have to reinstall `hbbs`.

**Reply by fufesou** [reply]
Please also backup the files in `/var/lib/rustdesk-server` before reinstall.

I have no idea what has happened on the machine.

**Comment by fufesou** [comment]
@mesuttavukcu Hi, how is this issue going?

**Reply by mesuttavukcu** [reply]
no, I'm reinstall and same problem continiu

**Reply by fufesou** [reply]
Still zero byte files in `/var/lib/rustdesk-server/static` and no files in `/usr/share/rustdesk-server/static`?

**Reply by mesuttavukcu** [reply]
I'm now format linux machine and reinstall. Problem is fixed. Thanks

**Comment by mesuttavukcu** [comment]
I have encountered a problem after reinstalling the server. My security key was renewed, and I created new clients that can connect without any issues. However, more than 200 devices with the old security key still appear on my active device list. Unfortunately, I am unable to delete them. Even though these devices are online, I cannot connect to them to uninstall RustDesk.

I urgently need assistance with this issue. Most importantly, for security reasons, no client with an incorrect security key should be able to connect to my server. I kindly request an immediate solution to this problem. Since I cannot clean up these devices, I am unable to provide support, and I also cannot identify which devices are problematic.

Thank you in advance for your urgent assistance.
